### ISSUE TITLE
[Suggestion] Let Silk moth hive be affected by dispensor.

### ISSUE THREAD

**Issue by Flyvee** [quiqueck/BetterEnd#153]
### Description

Currently the silk moth hive is only manual. So it would be nice if shears inside a dispensor can affect the silk moth hive the same way it does for a bee hive.
I would love to be able to automate this farm:
![2023-02-19_22 39 54](https://user-images.githubusercontent.com/30793646/219974091-5c325bab-e65d-4105-9f4a-d5bc7f15a305.png)

**Comment by rruy-dev** [comment]
Yes!! Please Please add this feature to and please backport it to v1.20.1 so I can enjoy the feature in Fantasy MC Fabric.

Would be awesome if the Silk Moth side of this setup could auto harvest
![2024-02-15_19 09 51](https://github.com/quiqueck/BetterEnd/assets/159833521/2b5eb0a6-76ad-4327-9f4c-d1e4dd3b8c2b)


Love the mods btw, adds so many cool new things.
